config/routes.rb

```diff
@@ -3,5 +3,7 @@
 
   devise_for :users, controllers: {sessions: 'sessions', registrations: 'registrations'}
 
-  resources :books
+  resources :books, :constraints => { :format => 'html' }
+  resources :authors, only: [:index], :constraints => { :format => 'html' }
+  resources :categories, only: [:index], :constraints => { :format => 'html' }
```

Comment:
2 comments:
- only index? no create? or new?
- what is the purpose of the constraints?

Reply:
- `create` and `new` are done in book's controller

- 
![image](https://user-images.githubusercontent.com/29805061/87099685-b481df00-c24a-11ea-88ae-e3ed5a4a3ed2.png)

Reply:
- then why do need the controllers for? index is not a good answer for this question
- I don't see a necessity to this.